### PR TITLE
snap: skip go version check

### LINKED ISSUE #265: snap: launchpad: curl: command not found
Could not build snap in launchpad

```
Building runtime 
INFO: yq was not found, installing it
.ci/install-yq.sh: line 56: curl: command not found
.ci/install-yq.sh: line 62: curl: command not found
chmod: cannot access '/build/kata-containers/parts/runtime/go/bin/yq': No such file or directory
ERROR: Cannot not get /build/kata-containers/parts/runtime/go/bin/yq executable
make: /build/kata-containers/parts/runtime/go/bin/yq: Command not found
golang.mk:20: *** "ERROR: cannot determine minimum golang version".  Stop.
Failed to run 'override-build': Exit code was 2.
```

### PR DESCRIPTION
**Pull request by devimc**
Right golang version is installed before building kata-containers, skip go
version check to avoid including extra build dependencies.

fixes #265

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

File: snap/snapcraft.yaml.in [hunk 1]
```diff
@@ -25,12 +25,13 @@ parts:
       unset GOROOT
       export GOPATH=$(realpath ../go)
       cd ${GOPATH}/src/github.com/${SNAPCRAFT_PROJECT_NAME}/runtime
-      QEMU_ARCH=$(arch)
```
Comment (jodh-intel) [inline snap/snapcraft.yaml.in]: Nit: This unused variable (?) removal should really be on a separate commit as it's unrelated to the version check.

**Comment by devimc** [comment]
@jodh-intel change applied, thanks

**Comment by jodh-intel** [comment]
Thanks @devimc.

lgtm
